context/spec/010-nextjs-16-feature-adoption/technical-considerations.md

```
@@ -0,0 +1,172 @@
+### 2.2 `"use cache"` Migration — IGDB Search Handler
+
+**File:** `savepoint-app/data-access-layer/handlers/igdb/igdb-handler.ts`
+
+Current pattern (lines 25–51):
+```
+unstable_cache(fn, [keys], { revalidate: 300, tags: ["igdb:search", `igdb:search:${query}`] })
+```
+
+Migration:
+- Replace `getCachedIgdbSearch` wrapper with an async function using `'use cache'` directive
+- Use `cacheTag("igdb:search", `igdb:search:${normalizedQuery}`)` for tag-based invalidation
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)